backend/src/client-sign/client-sign.controller.ts

```diff
@@ -0,0 +1,16 @@
+import { Body, Controller, Get, Param, Post, Res } from '@nestjs/common';
+import { ClientSignatureBody } from '../dto/register.dto';
+import { ClientSignService } from './client-sign.service';
+
+@Controller()
+export class ClientSignController {
+    constructor(private readonly clientSignService: ClientSignService) {}
+
+    @Post('client/signature')
```

Comment:
In the PoC this was implemented on the server but for the actual implementation, it should be implemented client side. "ethers" dependency works in the browser too.

Reply:
I see the implementation is also in the client. We can remove this backend endpoint once completed on the client.

Reply:
Client sign removed from the backend